ArduPlane/quadplane.cpp

```diff
@@ -1163,7 +1163,7 @@ bool QuadPlane::assistance_needed(float aspeed)
         return false;
     }
     
-    uint32_t max_angle_cd = 100U*assist_angle;
+    int32_t max_angle_cd = 100U*assist_angle;
```

Comment:
You do want this to be a unsigned type, or take `labs(assist_angle)` which would be the proper usage. (It's a user parameter, it could be negative)